core/src/datayoga_core/step.py

```diff
@@ -11,14 +12,14 @@
 logger = logging.getLogger("dy")
 
 
-class Step():
-    def __init__(self, id: str, block: Block, concurrency=1):
-        self.id = id
+class Step:
+    def __init__(self, step_id: str, block: Block, concurrency=1):
+        self.id = step_id
```

Comment:
shouldn't it be step_id as well?

Reply:
Not necessary cuz it's member of the object.